webservice/template/course.tmpl

```diff
@@ -88,7 +88,7 @@
                                                     <span class="badge {{if DEADLINEDUE .Deadline}}badge-dark{{ else}} badge-secondary{{end}}">{{$C.Code}}</span>
                                                 </h5>
                                                 <p{{if DEADLINEDUE .Deadline}} class="text-danger"{{end}}>
-                                                    <strong>Deadline: </strong> {{PRETTYTIME .Deadline}}</p>
+                                                    <strong>Deadline: </strong> {{PRETTYTIME .Deadline}} <i>(Norwegian time!)</i></p> <!-- TODO time-norwegian -->
```

Comment:
Remove "(Norwegian time!)"

Reply:
Christopher asked for this specific, so nah